src/software/control/src/chip/mod.rs

```diff
@@ -130,4 +143,17 @@ impl Chip {
             }
         }
     }
+
+    pub fn ongoing_alarms_sorted(&self) -> Vec<(&AlarmCode, &AlarmTrap)> {
+        let mut vec_alarms = self
+            .ongoing_alarms
+            .iter()
+            .collect::<Vec<(&AlarmCode, &AlarmTrap)>>();
+
+        // TODO: maybe there is a way to sort reverse?
```

Comment:
Maybe something with https://doc.rust-lang.org/std/cmp/enum.Ordering.html#method.reverse

Like `alarm1.alarm_priority.cmp(&alarm2.alarm_priority).reverse()`?

Reply:
Mh, that's nice, I found other solutions that were way worse and I didn't take the time to check.